lib/internal/worker.js

```diff
@@ -265,6 +265,7 @@ class Worker extends EventEmitter {
     setupPortReferencing(this[kPublicPort], this, 'message');
     this[kPort].postMessage({
       argv,
+      execArgv: options.execArgv,
```

Comment:
Does this affect all workers, not just the one for hooks?

Reply:
If InternalWorker is only created for hooks, the answer is yes

Reply:
This does seem odd, especially when `execArgv` is supposed to default to the parent for all `Worker` cases. I would also have expected a change to the consumption API too. Where is this consumed internally that this change is needed?